src/celengine/stardb.cpp

```diff
@@ -60,8 +61,8 @@ constexpr inline float STAR_OCTREE_ROOT_SIZE   = 1000000000.0f;
 constexpr inline float STAR_OCTREE_MAGNITUDE   = 6.0f;
 //constexpr const float STAR_EXTRA_ROOM        = 0.01f; // Reserve 1% capacity for extra stars
 
-constexpr inline std::string_view FILE_HEADER            = "CELSTARS"sv;
-constexpr inline std::string_view CROSSINDEX_FILE_HEADER = "CELINDEX"sv;
+constexpr inline std::string_view FILE_IDENTIFIER            = "CELSTARS"sv;
```

Comment:
it seems that using `inline` makes sense for header files only and it's noop for cpp files.

Reply:
Yep, I tend to prefer to keep it though in case of needing to move this to a header file later.